client/my-sites/stats/stats-module/connected-list.js

```diff
@@ -0,0 +1,148 @@
+/**
```

Comment:
Are we giving up on GitHub ever fixing `jsx` syntax highlighting and converting to `js`? Maybe it's about time 😀